spec/factories/courses.rb

```diff
@@ -209,5 +214,17 @@
     trait :skip_validate do
       to_create { |instance| instance.save(validate: false) }
     end
+
+    trait :unpublished_with_primary_maths do
+      transient do
+        identifier { "unpublished_with_primary_mathematics" }
+      end
+
+      name { "#{identifier} course" }
```

Comment:
I realised after I did this that it should actually be named after the attribute, so we can track where it gets used it if shows up elsewhere.

```suggestion
      name { "#{identifier} course name" }
```